### DISCUSSION TITLE
Object Eraser Showcase

### DISCUSSION THREAD

**Discussion by deltheil** [finegrain-ai/refiners#395] [Announcements]
# We've created a new **[HuggingFace Space](https://huggingface.co/spaces/finegrain/finegrain-object-eraser)** showcasing our :sponge:  **Object Eraser pipeline** !

There are lots of AI object erasers available, but ours feels truly special: it doesn't require manual work (a text prompt is enough) and it will erase the object AND all its effects in your image (like shadows and reflections):

https://github.com/user-attachments/assets/0751d718-a8f2-4d2f-ada1-60c42abeb3a3

# 📣 Enjoyed it? Share Online!

We need your help to spread the word: feel free to share on your social networks! 🙏

**Comment by RunpuWei** [comment]
Thanks for your interesting work, I'm curious how you implement this feature, it's quite amazing.

**Reply by dnsbrl** [reply]
Hey, thanks for the feedback! We trained our own model to achieve this level of quality (ensuring we're able to erase not just the object but also its effects on the scene like shadows or reflections). We're considering open sourcing it - stay tuned!

**Comment by MeChun** [comment]
intently waiting for the open source Finegrain Object Eraser
